lighting-estimation-explainer.md

```diff
@@ -0,0 +1,91 @@
+
+WebXR API supports up to 9 SH coefficients per RGB color component, for a total of 27 floating point scalar values.  This enables the level 2 (3rd) order of details.  If a platform can not supply all 9 coefficients, it can pass 0 for the higher order coefficients resulting in an effectively lower frequency reproduction.  
+
+This "SH probe" format is used by most modern rendering engines, including Unity, Unreal, and Threejs.
+
+## Shadows
+
+When an HDR Cube Map texture is available, shadows only have to consider occlusion of other rendered objects in the scene.
+
+When a HDR Cube Map texture is not available, or the typical soft shadow effects of image based lighting are too costly to implement, the XRLightProbe.primaryLightDirection and XRLightProbe.primaryLightIntensity can be used to render shadows cast by the most prominent light source.
+
+## Security Implications
+
+### XRLightProbe
+
+Only XRLightProbe.indirectIrradiance is guaranteed to be available either due to user privacy settings or the capabilities of the platform.
+
+XRLightProbe returns sufficient information to render objects that appear to fit into their environment, with highly diffuse surfaces or high frequency normal maps which would result in a wide NDF (normal distribution function).  Highly polished objects may be represented with a non-physically based illusion of glossiness with a specular highlight effect sensitive only to the primary light direction.  Reflections will be unable to reproduce detailed images of the environment without an XRReflectionProbe.
+
+The lighting estimation returned by the WebXR API explicitly describes the real world environment in proximity to the user.  By default, only low spatial frequency and low temporal frequency information should be returned by the WebXR API.  Even when a platform can directly produce higher spatial and temporal frequency information, the browser must apply a low pass filter with an aim to mitigate the risk of untrusted content identifying the geolocation of the user or of profiling their environment.
+
+Combined with other factors, such as the user's IP address, even the low frequency information returned with XRLightProbe increases the fingerprinting risk.  The XRLightProbe should only be accessible during an active WebXR session.
+
+### XRReflectionProbe
+
+XRReflectionProbe should only be accessible with a permissions prompt equivalent to requesting access to the camera and microphone.  XRReflectionProbe enables efficient and simple to implement image based lighting.  PBR shaders can index the mip map chain of the environment cube to reduce the memory bandwidth required while integrating multiple samples to match wider NDF's.
+
```

Comment:
I don't understand the reference to **and microphone** here. Why not just camera?

Reply:
I was imagining similar UX as requesting microphone permission, not necessarily relating the kind of sensor.  It seems that this analogy is adding more confusion than helping, so perhaps I should remove the "microphone" reference.